### ISSUE TITLE
Open Access papers

### ISSUE THREAD

**Issue by philippbayer** [gedankenstuecke/scihub_analysis#4]
A cursory glance at the data reveals quite a few open access papers, for PLOS Biology:

```
>grep -c pbio *
dec2015.tab:1722
feb2016.tab:3154
jan2016.tab:2486
nov2015.tab:912
etc.
```

or PLOS ONE:

```
>grep -c pone *
dec2015.tab:299
feb2016.tab:532
jan2016.tab:363
nov2015.tab:164
```

It would be interesting to see how often people use SciHub to get access to content they already have, for example because they use SciHub as the direct replacement for Pubmed/Google Scholar. One could either just check for the most common OA journals, or use the Mendeley API to query OA-status of each paper (cumbersome), and then try to find any correlations - I expect there to not be any, I'd assume a portion of the rich and poor alike use SciHub as replacement.

**Comment by gedankenstuecke** [comment]
Good idea to look into this. Won't have any time to compile that data until at least next week though. So feel free to go ahead if you want! Happy to merge whatever comes this way 😃
